src/containers/flows/MintDropNFT/index.tsx

```diff
@@ -345,9 +371,7 @@ const MintDropNFTFlowContainer = () => {
 		) {
 			// Fetch minted count periodically
 			timer = setInterval(async () => {
-				const sold = await zSaleInstance.getNumberOfDomainsSold(
-					library.getSigner(),
-				);
+				const sold = await zSaleInstance.getNumberOfDomainsSold();
 				if (sold) setWheelsMinted(sold.toNumber());
 			}, 1000);
```

Comment:
I think this number will need to be closer to `5000` or even `7000` - every call to `getNumberOfDomainsSold` will cost a request through Infura, I think one per second would push us over our daily limit.